roles/percona-server/tasks/main.yml

```diff
@@ -5,6 +5,20 @@
 - name: create mysql config directory
   file: path=/etc/mysql/conf.d state=directory
 
+- name: check to see if /var/log/mysql/mysql.err exists
+  stat: path=/var/log/mysql/mysql.err
+  register: mysql_error_log
+
+- name: change /var/log/mysql/mysql.err to mode 644
+  file: dest=/var/log/mysql/mysql.err mode=0644
+  when: mysql_error_log.stat.exists == True
+
+- name: set UMASK variable for mysqld_safe
+  lineinfile: dest=/etc/default/mysql create=yes
+              line="UMASK=0644"
+  notify:
+    - restart mysql server
```

Comment:
No mysqld_safe is really big. But the file does already exist. It has some timeout values.